tests/gene-comparison-tool.spec.ts

```diff
@@ -27,4 +27,15 @@ test.describe('specific viewport block', () => {
     const dropdown = page.locator('#subCategory');
     await expect(dropdown).toHaveText('Targeted Selected Reaction Monitoring (SRM)');
   });
+
+  test('invalid gene results in a 404 redirect', async ({ page }) => {
+    // go to invalid ENSG page
+    await page.goto('/genes/ENSG00000000000');
+
+    // expect a title "to contain" a substring.
+    await expect(page).toHaveTitle('Agora');
```

Comment:
Not sure if I like having the title just be 'Agora' but that could be something to change down the line.